Prox/Prox/Widgets/LocationLoadingView.swift

```diff
@@ -11,6 +11,7 @@ fileprivate func createCircle(color: CGColor, frame: CGRect) -> CAShapeLayer {
     circle.path = CGPath(ellipseIn: frame, transform: nil)
     circle.fillColor = color
     circle.frame = frame
+    circle.opacity = 0.2
```

Comment:
Changed the default opacity to 0.2 because when we came back to the foreground the blue was dark before starting the animation again. This makes the transition less jolting.